app/generator/index.js

```diff
@@ -0,0 +1,24 @@
+const PdfPrinter = require('pdfmake')
+const moment = require('moment')
+const fonts = require('./fonts')
+const styles = require('./styles')
+const generateContent = require('./content')
+const createLog = require('./create-log')
+const publish = require('./publish')
+const printer = new PdfPrinter(fonts)
+
+const generateStatement = async (statement) => {
```

Comment:
thinking this func should have `try-catch`

Reply:
It's already in a try catch at a higher level for processing of the original message that will capture the error.  What would be the advantage of a lower level catch that just throws the error again?

Reply:
Truesy -- ignore first comment